arangod/Aql/Function.h

```diff
@@ -55,7 +55,10 @@ struct Function {
 
     /// @brief exclude the function from being evaluated during AST
     /// optimizations evaluation of function will only happen at query runtime
-    NoEval = 8
+    NoEval = 8,
+    
+    /// @brief is this function position dependent
+    NonMoveable = 16
```

Comment:
Right now, the new enum value is not used anywhere. I understand this may be used in the future, but I suggest we introduce it only when it is actually being used.
```suggestion
```